bassist/bassist/parser/log_file/debs_stdout.py

```diff
@@ -2,10 +2,11 @@
 # See the file LICENSE for copying permission.
 from . import common
 from ...systems import package
-from results import store_debs
-from run_sql import execute_sql
+
+debs = []
```

Comment:
Most of the code is part of the other PR #17, which was merged. Please change this PR so it only includes the commits relevant to the DB duplicates issue.